Library/Formula/invada-studio-plugins-lv2.rb

```diff
@@ -0,0 +1,20 @@
+class InvadaStudioPluginsLv2 < Formula
+  desc "LV2 audio plugins suite"
+  homepage "https://launchpad.net/invada-studio/"
+  url "https://launchpad.net/invada-studio/lv2/1.2/+download/invada-studio-plugins-lv2_1.2.0-nopkg.tgz"
+  version "1.2.0"
+  sha256 "c6cac7c32effc6b3052e3b017133244f385ef8e053147859d88eae6facaf7d12"
+  head "lp:invada-studio/lv2", :using=>:bzr
+
+  depends_on "lv2"
```

Comment:
That would be possible, but I don't think that this is an modular approach. Many resources might be added to the `lv2` each with different dependencies, different versions (stable/head) making the `lv2` formula very fragile and subject of frequent change. 

Currently there is already a similar formula in homebrew btw: [mda-lv2](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/mda-lv2.rb)